pywavefront/obj.py

```diff
@@ -62,15 +62,18 @@ def consume_vertices(self):
                 )
 
             self.next_line()
-            if self.values[0] != "v":
+            if not self.values:
+                break
+
+            if self.values and self.values[0] != "v":
```

Comment:
By this point, we know we have `self.values` already right? In that case, can this be simplified to `if self.values[0]...`

Reply:
derp